### PR TITLE
Milestone4

### PR DESCRIPTION
**Pull request by sergiocortessat**

### CODE REVIEW

**Comment by sergiocortessat** [comment]
Hello TSE. Thanks for reviewing my request.  Deliver of Milestone 4.

As such I have implemented: 
-The Redux store keeps the information about the current books filter. 
- Created the CHANGE_FILTER action in actions/index.js
- Created a filter.js reducer which modifies filter in the Redux store. it defaults to All
- Implemented the CategoryFilter component
- Render the CategoryFilter component in the BooksList component.
- Implemented the handle interchange method which modifies the filter state in the Redux store. 
- Implemented a conditional method that decides which books should be shown in the table depending on the filter value from the Redux store.

File: src/Components/CategoryFilter.js [hunk 1]
```diff
@@ -0,0 +1,7 @@
+/* eslint-disable react/prop-types */
```
Comment (bellom) [inline src/Components/CategoryFilter.js]: With the ESLint inline comment, you are indirectly changing the linter's rules by disabling these rules, let's fix the codebase instead.

Perhaps adding the prop-types would fix the error.

File: src/Containers/BooksList.js [hunk 2]
```diff
@@ -1,19 +1,30 @@
+/* eslint-disable consistent-return */
 /* eslint-disable no-unused-vars */
```
Comment (bellom) [inline src/Containers/BooksList.js]: With the ESLint inline comment, you are indirectly changing the linter's rules by disabling these rules, let's fix the codebase instead.


If you are using VSCode, you can delete the inline comment and hover over the error(configure Eslint checks locally). And you will see your options. Click on the more info button that will take you to a GitHub repo where it is explained why you are getting this error. I find this feature very educational. Please, try to learn and implement the necessary rules so you don't have to keep this disabling line on top of each file. I believe you can do this! 💪😇

File: src/Containers/NavBar.js [hunk 3]
```diff
@@ -0,0 +1,32 @@
+/* eslint-disable max-len */
```
Comment (bellom) [inline src/Containers/NavBar.js]: With the ESLint inline comment, you are indirectly changing the linter's rules by disabling these rules, let's fix the codebase instead.

File: src/reducers/filter.js [hunk 4]
```diff
@@ -0,0 +1,12 @@
+// const initialState = 'ALL';
```
Comment (bellom) [inline src/reducers/filter.js]: You need to remove the commented code, It’s better to delete the commented code. If you need to review a previous version of your code, use git history.

**Review by bellom — CHANGES_REQUESTED** [review]
## Hi @sergiocortessat  :clap: 

### Awesome job so far!! :+1: 

#### You have done wonderfully well, you passed `almost` all the checklist for this project :+1: just a few issues left to work on............ :slightly_smiling_face: 

# Status: Changes Required! ♻️ 

**Kindly check my comments under this review;**

### ♻️ Kindly re-submit for review after making all those changes :slightly_smiling_face: Thumps -up await you.....

**Best**

**Comment by sergiocortessat** [comment]
Hello TSE

I have removed all inline comments requested

**Review by voscarmv — APPROVED** [review]
# Project status: Approved 💯 ✨ 😎 👍 
### You may merge this branch and move forward ⛙🛣️ 

### Questions or comments? Reach me on [Twitter](https://twitter.com/voscarmv) or [LinkedIn](https://linkedin.com/in/oscar-mier) 🧑‍💻🗨️🧑‍💻
# Happy Coding! 🧠 ⌨️ 🖱️ 🖥️

**Pull request merged**